xgcm/grid.py

```diff
@@ -758,6 +765,97 @@ def max(
             keep_coords,
         )
 
+    def transform(
+        self,
+        da,
+        target,
+        target_data=None,
+        target_dim=None,
+        method="linear",
+        mask_edges=False,
+    ):
+        # check optional numba dependency
+        if numba is None:
+            raise ValueError(
```

Comment:
This should be an `ImportError`.

Reply:
Ah ok, I was trying to check what xarray uses, but the error handling is quite convoluted over there. Will fix

Reply:
Fixed.